<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneWidget</class>
 <widget class="QWidget" name="SceneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>790</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="mpObjectCreateBox">
     <property name="title">
      <string>Create Object</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mpLabelName_1">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="mpLineEditName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mpLabelType">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="mpComboBoxType"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="mpPushButtonCreateObject">
        <property name="text">
         <string>Create Object</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="mpPushButtonDeleteAllObjects">
        <property name="text">
         <string>Delete All Objects</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mpGroupBoxLoad">
     <property name="title">
      <string>Load a object set from a file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mpLoadPathLineEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="mpPushButtonLoadSet">
        <property name="text">
         <string>Load object set from file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="mpGroupBoxSave">
     <property name="title">
      <string>Write out current object set to file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mpSavePathLineEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="mpPushButtonSaveSet">
        <property name="text">
         <string>Save object set to file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="mpObjectSettingsBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Object Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,0,0,0,0,0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="mpLabelName_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="7">
       <widget class="QComboBox" name="mpComboBoxName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mpLabelChessboardSize">
        <property name="text">
         <string>rows:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mpLabelColorR_1">
        <property name="text">
         <string>R:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxColorR">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="mpLabelColorG_1">
        <property name="text">
         <string>G:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="mpLabelColorB_1">
        <property name="text">
         <string>B:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxColorB">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="8">
       <widget class="QPushButton" name="mpPushButtonDeleteObject">
        <property name="text">
         <string>Delete Object</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxHeight">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="mpLabelHeight">
        <property name="text">
         <string>height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mpLabelLenght">
        <property name="text">
         <string>length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="mpLabelWidth">
        <property name="text">
         <string>width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxWidth">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mpLabelRadius">
        <property name="text">
         <string>radius</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxRadius">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxThickness">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QCheckBox" name="mpWireFrameCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>wire frame</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QSpinBox" name="mpSpinBoxChessboardColumns">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>columns:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="mpSpinBoxChessboardRows">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxColorG">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="mpDoubleSpinBoxLength">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="">
     <property name="title">
      <string>Rigid body</string>
     </property>
     <layout class="QVBoxLayout" name="mpRigidBodyWidgetLayout"/>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="">
     <property name="title">
      <string>Rigid body visualization</string>
     </property>
     <layout class="QVBoxLayout" name="mpRigidBodyVisualizationWidgetLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
